src/configcatclient.cpp

```diff
@@ -218,9 +219,11 @@ EvaluationDetails ConfigCatClient::_getValueDetails(const std::string& key, Valu
         for (auto keyValue : *settings) {
             keys.emplace_back(keyValue.first);
         }
-        LogEntry logEntry(logger, LOG_LEVEL_ERROR);
-        logEntry << "Value not found for key " << key << ". Here are the available keys: " << keys
-                 << " Returning defaultValue: " << defaultValue << " .";
+        LogEntry logEntry(logger, LOG_LEVEL_ERROR, 1001);
+        logEntry <<
+            "Failed to evaluate setting '" << key << "' (the key was not found in config JSON). "
+            "Returning the `defaultValue` parameter that you specified in your application: '" << defaultValue << "'. "
+            "Available keys: [" << keys << "].";
```

Comment:
[] is not needed here because LogEntry decorates the string vectors with [].
```suggestion
            "Available keys: " << keys << ".";
```